web/src/components/dialog/NoScriptDialog.svelte

```diff
@@ -0,0 +1,44 @@
+<script lang="ts">
+    import { onMount } from "svelte";
+    import SmallDialog from "./SmallDialog.svelte";
+
+    let noscriptElem: HTMLElement;
+    onMount(() => noscriptElem.remove());
+</script>
+
+<noscript bind:this={noscriptElem} style="display: contents">
+    <div id="nojs-ack">
+        <SmallDialog
+            id="nojs-dialog"
+            meowbalt="error"
+            bodyText={
+                "cobalt uses javascript for api requests and interface interactions, but it has been disabled, "
+                + "or your browser doesn't support it. you can navigate around cobalt, but most functionality "
+                + "will not work."
```

Comment:
`cobalt uses javascript for api requests and ui interactions, but it's not available in your browser. you can still navigate around cobalt, but most functionality won't work.`